api/app/serializers/v1/text_section_serializer.rb

```diff
@@ -3,19 +3,24 @@ class TextSectionSerializer < ManifoldSerializer
 
     include ::V1::Concerns::ManifoldSerializer
 
-    typed_attribute :text_slug, NilClass
-    typed_attribute :text_title, NilClass
-    typed_attribute :name, NilClass
-    typed_attribute :social_image, NilClass
-    typed_attribute :source_identifier, NilClass
-    typed_attribute :kind, NilClass
+    typed_attribute :text_slug, Types::String
+    typed_attribute :text_title, Types::String
+    typed_attribute :name, Types::String.optional
+    typed_attribute :social_image, Types::Serializer::URL
```

Comment:
Do you happen to know what type the `social_image` is It looks like it's just a single URL, but I'm not confident.

Reply:
Yes, It's a single URL. See `project.social_image`. The TextSection delegates the request to the Text, and the Text delegates it to the project.

Reply:
Thank you!